qaframework-bdd-tests/src/test/resources/features/refapp-2.x/02-login/verifyLogin.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User based login
+
+@selenium
+@defaultLogin
+Scenario Outline: User Based login verification
+   When user verifies modules available on home page
+   And user verifies modules available on homepage after login as clerk
+   And user verifies modules available on home page after login as Doctor
+   And user verifies modules available on home page after login as Nurse
+   And user verifies modules available on home page after login as sysadmin
```

Comment:
isn't it better to have the right work for `sysadmin`?

Reply:
Would like to keep it the way it was initially.

Reply:
system adminstration .